packages/gestalt/src/DropdownLink.js

```diff
@@ -41,6 +41,7 @@ type Props = {|
   onClick?: ({|
     event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement>,
     dangerouslyDisableOnNavigation: () => void,
+    mobileOnDismissStart: () => void,
```

Comment:
Can you add documentation about this new field?

Reply:
DONE